### PR TITLE
cleaned up so that `npm test` runs without errors

### PR DESCRIPTION
**Pull request by pfrollo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "license": "GPL-3.0",
   "scripts": {
     "lint": "node_modules/eslint/bin/eslint.js src/** test/**",
-    "test": "mocha --compilers js:babel-register ./test/*.js",
+    "test": "mocha ./test/*.js",
```
Comment (richard3d) [inline package.json:10]: The `--compilers js:babel-register` portion of this line can actually be replaced with `--require babel-core/register` and then you do not have to switch from ES to CommonJS modules either (I'd prefer to keep ES in this case, so maybe revert those changes). Keep `mocha` in the package.json of course and `.idea` is fine in `.gitignore`

**Review by richard3d — CHANGES_REQUESTED** [review]
See comment on package.json

File: package.json (hunk 1 above)
Comment (pfrollo) [reply package.json]: fixed

File: src/index.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ export class StatEngine {
 
     const incident = {
       id: params.id,
-      timestamp: params.timstamp || moment.utc().format(),
+      timestamp: params.timestamp || moment.utc().format(),
```
Comment (pfrollo) [inline src/index.js:33]: @richard3d - none of the lambdas call this api with timstamp or timestamp so I fixed what appeared to be a typo

**Pull request merged**